datamol/convert.py

```diff
@@ -317,7 +391,7 @@ def from_df(
     mol_column: str = None,
     conserve_smiles: bool = False,
     sanitize: bool = True,
-) -> List[Chem.rdchem.Mol]:
+) -> List[dm.Mol]:
     """Convert a dataframe to a list of mols.
```

Comment:
Out of scope, and just curious, but isn't the function name `to_df` the reverse of the function description `Convert a dataframe to a list of mols`

I would expect the following:
- `to_df`: Convert a **list of mols** to a **data frame**.
- `to_list`: Convert a **dataframe** to a **list of mols**. (which the method seems to be doing here no?)

Looking at it more closely, the first argument in the docstring is `df` and the first argument in the function parameters is `mols`

something to double check and change according to what we want here I think

Reply:
I think `from_df` does this. I will complement the docstring to xref those 2 functions to each others.

Reply:
Ok, the tldr point here was that there is a mismatch between the function name and docstring

Reply:
nvm all of this, I misread the PR/codebase this morning